installers/linux-shared/after-remove.sh.erb

```diff
@@ -17,9 +17,9 @@
 -%>
 
 <% if type == 'rpm' -%>
-    rm -rf /var/lib/<%= name %> /var/log/<%= name %> /etc/defaults/<%= name %>
+    rm -rf /var/lib/<%= name %> /var/log/<%= name %>
 <% elsif type == 'deb' -%>
     update-rc.d <%= name %> remove
-    rm -rf /var/lib/<%= name %> /var/log/<%= name %> /etc/defaults/<%= name %>
+    rm -rf /var/lib/<%= name %> /var/log/<%= name %>
```

Comment:
Compared to how the old installers were, this is dangerous (unless update-remove.sh is called properly elsewhere - haven't checked). The earlier script (in 16.6 deb installer) was:

```
if [ "$1" = "purge" ] ; then
    update-rc.d go-server remove >/dev/null || exit $?

    rm -fr /var/lib/go-server
    rm -fr /etc/go
    rm -fr /usr/share/go-server
    rm -fr /var/run/go-server
    rm -fr /var/log/go-server
fi
```